README.md

```diff
@@ -22,6 +28,20 @@ The intention of Safe Harbor language provided by disclose.io is for it to be fo
 
 Policies missing any of the core tenets above, but that still contain a good-faith statement committing not to pursue legal action on security researchers, meet the criteria for **Partial Safe Harbor**.
 
+### Disclosure types
+
+- **Coordinated Disclosure**: A researcher can share details of the vulnerability after a fix has been applied and the program owner has provided permission to disclose or after a clearly-stated time has passed from submission, whichever is sooner;
+- **Discretionary Disclosure**: The researcher or the program owner can request mutual permission to share details of the vulnerability after approval is explicitly received; or
```

Comment:
Is it worth providing that this may be something that is a process that can happen in whole or in part?

Reply:
I'm not totally following - Can you give an example?

In general, the language prefers strict definitions in order to avoid ambiguity.

Reply:
Sure, so this would be in an instance where a vulnerability disclosure contains certain privileged information that for whatever reason is not ready for public consumption (e.g. you can disclose X if you omit detail Y). Would this fall within this category?

Reply:
Yes, in coordinated and discretionary disclose - provided that the communication channel between the finder and recipient is active - it's customary for there to be a conversation about what is wise to include or omit from public disclosure.